performance_manager/lib/l0_gtfs_static_table.py

```diff
@@ -273,15 +277,21 @@ def process_static_tables(db_manager: DatabaseManager) -> None:
             load_parquet_files(static_tables, folder)
             transform_data_tables(static_tables)
             insert_data_tables(static_tables, db_manager)
-        except Exception as exception:
-            individual_logger.log_failure(exception)
-        else:
+
             update_md_log = (
                 sa.update(MetadataLog.__table__)
                 .where(MetadataLog.pk_id.in_(ids))
                 .values(processed=1)
             )
             db_manager.execute(update_md_log)
             individual_logger.log_complete()
+        except Exception as exception:
+            update_md_log = (
+                sa.update(MetadataLog.__table__)
+                .where(MetadataLog.pk_id.in_(ids))
+                .values(processed=1, process_fail=1)
+            )
+            db_manager.execute(update_md_log)
+            individual_logger.log_failure(exception)
```

Comment:
update to metadata log to indicate failure to load file

Reply:
👍